src/cloudant/design_document.py

```diff
@@ -27,7 +27,7 @@ class DesignDocument(Document):
     :class:`~cloudant.document.Document`.  A DesignDocument object is
     instantiated with a reference to a database and
     provides an API to view management, list and show
```

Comment:
After reviewing the link below and creating show/list functions in test design documents, I can access the `_list` and `_show` endpoints in both CouchDB and Cloudant:
http://docs.couchdb.org/en/1.6.1/couchapp/ddocs.html